src/zap/zapinfo.cpp

```diff
@@ -450,6 +450,24 @@ void ZapInfo::CompileMethod()
         return;
     }
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
+    if (methodAttribs & CORINFO_FLG_JIT_INTRINSIC)
+    {
+        // Skip generating hardware intrinsic method bodies.
+        //
+        // The actual method bodies are only reachable via reflection/delegates, but we don't know what the
+        // implementation should do (whether it can do the actual intrinsic thing, or whether it should throw
```

Comment:
I think the former issue is the fundamental one - there's really no point in consuming crossgen time or executable code space with these. We *could* generate bodies for the ones in `Sse` and `Sse2` since they're in the baseline, but it wouldn't be very useful.

Reply:
I agree.

I also think generating the method bodies for SSE/SSE2, while it could be done, also needs further consideration in how it interacts with `COMPlus_EnableSSE` (which may be particularly impactful when this is eventually extended to support non S.P.Corelib libraries).